src/content/docs/en/pages/main-menu/reference/observe/real-time-events/real-time-events.mdx

```diff
@@ -41,261 +41,271 @@ You can use Real-Time Events to:
+| Proxy Host | Hostname being proxied. Example: `storage.googleapis.com:443` |
+| Proxy Status | HTTP error status code or origin when no response is obtained from the upstream. Example: `520`. **In case of cache, the response is `-`.** |
+| Proxy Upstream | Origin (upstream) address. In some cases, the L2 origin can be Image Processor (IMS) to process the image and then cache it. Example: `ims_http` |
+| Reference Error | Reference ID of the request. Generated when the status code is `4xx` or `5xx`. Example: `#AECFE66100000000C947B9B3B3BFBE46FFFFFFFF9401` |
+| Remote Addr | IP address of the origin that generated the request. Example: `127.0.0.1` |
+| Remote Port | Port of the origin that generated the request. Example: `8080` |
+| Request Length | Request length, including request line, headers, and body. This field is the result of a sum. Example: `167` |
+| Request Method | HTTP request method. Example: `GET` or `POST` |
+| Request Time | Request processing time elapsed since the first bytes were read from the client with resolution in milliseconds. This field is the result of a sum. Example: `1.19` |
+| Request Uri | URI of the request made by the end user, without the host and protocol information and with arguments. Example: `/v1?v=bo%20dim` |
+| Scheme | Request scheme. Example: `HTTP` or `HTTPS` |
+| Sent HTTP Content Type | `Content-Type` header sent in the origin’s response. Example: `text/html; charset=UTF-8` |
+| Server Protocol | Request protocol. Example: `HTTP/1.1`, `HTTP/2.0`, `HTTP/3.0` |
+| Solution | Identifier of your edge application. Example: `1321` |
+| Status | HTTP status code of the request. Example: `200` |
+| TCP info RTT | Round-Trip Time (RTT) measured by the edge for the user. Available on systems that support the TCP_INFO socket option. Example: `72052` |
+| Upstream Bytes Received | Number of bytes received by the origin’s edge if the content isn’t cached. Example: `8304` |
+| Upstream Cache Status | Status of the local edge cache. Can be: `MISS`, `BYPASS`, `EXPIRED`, `STALE`, `UPDATING`, `REVALIDATED`, `HIT`, or `-` |
+| Upstream Connect Time | Time it takes for the edge to establish a connection with the origin in milliseconds. In the case of TLS, it includes time spent on handshake. Example: `0.123`. **`0` in case of KeepAlive and `-` in case of cache.** |
+| Upstream Header Time | Time it takes for the edge to receive the response header from the origin in milliseconds. Example: `0.345`. **In case of cache, the response is `-`**. |
+| Upstream Response Time | Time it takes for the edge to receive a default response from the origin in milliseconds, including headers and body. Example: `0.876`. **In case of cache, the response is `-`**. |
+| Upstream Status | HTTP status code of the origin. If a server can't be selected, the variable keeps the 502 (Bad Gateway) status code. Example: `200`. **In case of cache, the response is `-`**. |
```

Comment:
```suggestion
| Upstream Status | HTTP status code of the origin. If a server can't be selected, the variable keeps the `502 Bad Gateway` status. Example: `200`. **In case of cache, the response is `-`**. |
```